policyengine_us/parameters/gov/contrib/congress/golden/fisc_act/family_income_supplement/phase_out/joint.yaml

```diff
@@ -0,0 +1,20 @@
+description: Representative Jared Golden proposed a family income supplement which is phased out at this rate for joint filers, based on adjusted gross income.
```

Comment:
technically $16.67 for each $1,000. if approximating with a smooth phase out in this PR could you file an issue to change?

Reply:
filed https://github.com/PolicyEngine/policyengine-us/issues/5621